packages/openai-compatible/src/openai-compatible-chat-language-model.ts

```diff
@@ -171,6 +171,11 @@ export class OpenAICompatibleChatLanguageModel implements LanguageModelV1 {
       seed,
       ...providerMetadata?.[this.providerOptionsName],
 
+      reasoning_effort:
+        providerMetadata?.[this.providerOptionsName]?.reasoningEffort ??
+        providerMetadata?.['openai-compatible']?.reasoningEffort ??
+        this.settings.reasoningEffort,
+
```

Comment:
why bother about the setting?

Reply:
sure either way works. my thinking is that people will need to migrate if we make it a setting

Reply:
i think maybe reducing migrations is a good idea, will remove

Reply:
changed, thanks 🙏